src/main/java/application/chunks/chunk/ChunkType.java

```diff
@@ -0,0 +1,16 @@
+package application.chunks.chunk;
+
+import javafx.scene.layout.GridPane;
+
+/**
+ * Interface for variable Chunk.
```

Comment:
Single chunk that contains {@link BlockType} objects.
This will be used by {@link Chunks}.